templates/csharp/Skill/Skill/Startup.cs

```diff
@@ -72,7 +71,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IChannelProvider, ConfigurationChannelProvider>();
 
             // Register AuthConfiguration to enable custom claim validation.
-            services.AddSingleton(sp => new AuthenticationConfiguration { ClaimsValidator = new AllowedCallersClaimsValidator(sp.GetService<IConfiguration>()) });
+            services.AddSingleton(sp => new AuthenticationConfiguration
+            {
+                ClaimsValidator = new Microsoft.Bot.Connector.Authentication.AllowedCallersClaimsValidator(sp.GetService<IConfiguration>().GetSection("allowedCallers").Get<List<string>>())
```

Comment:
```suggestion
                ClaimsValidator = new AllowedCallersClaimsValidator(sp.GetService<IConfiguration>().GetSection("allowedCallers").Get<List<string>>())
```
As we removed the one of the project, it's not necessary to keep that reference